client/src/pages/Dialogs/Login.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component, Fragment } from "react";
+import { Dialog, Button } from "@material-ui/core";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import TextField from "@material-ui/core/TextField";
+import { withStyles, createMuiTheme } from "@material-ui/core/styles";
+import Fab from "@material-ui/core/Fab";
+import { ThemeProvider } from "@material-ui/styles";
+import axios from "axios";
+
+const styles = theme => ({
+  dialogPaper: {
+    minHeight: "73vh",
+    maxHeight: "73vh",
+    minWidth: "45vw",
+    maxWidth: "45vw",
+    display: "flex",
+    alignItems: "center"
+  },
+  formWidth: {
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  textFieldHeader: {
+    margin: 0,
+    fontWeight: 600
+  },
+  FormControl: {
+    width: "60%"
+  },
+  footerDialog: {
+    borderTop: "1px solid lightgray",
+    marginTop: "25px",
+    paddingTop: "10px",
+    width: "100%",
+    display: "flex",
+    justifyContent: "center"
+  },
+  footerLink: {
+    textDecoration: "none",
+    color: "#DF1B1B"
+  }
+});
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: "#e21919"
+    }
+  }
+});
+
+export default withStyles(styles)(
+  class Signin extends React.Component {
+    state = {
+      open: true,
+      data: {
+        email: "",
+        password: ""
+      }
+    };
+
+    handleToggle = () => {
+      this.setState({
+        open: !this.state.open
+      });
+    };
+
+    handleChange = name => ({ target: { value } }) => {
+      this.setState({
+        data: {
+          ...this.state.data,
+          [name]: value
+        }
+      });
+    };
+
+    handleSubmit = data => {
+      try {
+        axios
+          .post("http://localhost:3000/user/login", {
```

Comment:
because of the proxy, you don't need the localhost here - it should just be the the path (`/user/login`), so when you deploy you don't need to change hthis everywhere)